index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
Your effort is remarkable and you were close to make it however you need some changes to make it much better.